ibm_db2/metadata.csv

```diff
@@ -0,0 +1,50 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
I am trusting you here. However if there are area you are unsure. Feel free to ask. 
Also I am assuming we have all metrics listed here. I let you double check if you are unsure.